jwst/gain_scale/tests/test_gain_scale.py

```diff
@@ -0,0 +1,62 @@
+"""
+Unit tests for gain_scale correction
+"""
+
+from jwst.datamodels import RampModel
+from jwst.gain_scale.gain_scale import do_correction
+from jwst.gain_scale import GainScaleStep
+import numpy as np
+import pytest
+
+def test_correction(make_rampmodel):
+    """Make sure correct gain factor is applied
+    """
+    datmod = make_rampmodel(2, 2048, 2048)
+    output = do_correction(datmod, 
+                           gain_factor=datmod.meta.exposure.gain_factor)
+
+    assert(output.meta.cal_step.gain_scale == 'COMPLETE')
+    assert np.all(output.err == datmod.err*datmod.meta.exposure.gain_factor)
```

Comment:
Any reason not to test the output.data vs datmod.data too?

Reply:
Added